eve/workers/end2end/scripts/install-kind-dependencies.sh

```diff
@@ -74,30 +85,73 @@ stringData:
   mongodb-replica-set-key: $MONGODB_RS_KEY
 EOF
 
-kubectl create configmap \
-    --from-file=${DIR}/configs/mongodb_init_scripts mongodb-init-scripts \
-    --dry-run=client -o yaml | kubectl apply -f -
-
-helm upgrade --install dev-db bitnami/mongodb \
-    --version $BITNAMI_MONGODB_VER \
-    -f "${DIR}/configs/mongodb_options.yaml"  \
-    --set "volumePermissions.enabled=true" \
-    --set "persistence.storageClass=standard" \
-    --set "usePassword=true" \
-    --set "mongodbRootPassword=$MONGODB_ROOT_PASSWORD" \
-    --set "replicaSet.key=$MONGODB_RS_KEY" \
-    --set "extraEnvVars[0].name=MONGODB_APP_USERNAME" \
-    --set "extraEnvVars[0].value=$MONGODB_APP_USERNAME" \
-    --set "extraEnvVars[1].name=MONGODB_APP_PASSWORD" \
-    --set "extraEnvVars[1].value=$MONGODB_APP_PASSWORD" \
-    --set "extraEnvVars[2].name=MONGODB_APP_DATABASE" \
-    --set "extraEnvVars[2].value=$MONGODB_APP_DATABASE" \
-    --set "extraEnvVars[3].name=MONGODB_NAMESPACE" \
-    --set "extraEnvVars[3].value=default" \
-    --set "replicaSet.pdb.minAvailable.secondary=1" \
-    --set "replicaSet.pdb.minAvailable.arbiter=0" \
-    --set "replicaSet.replicas.secondary=0" \
-    --set "replicaSet.replicas.arbiter=0"
-
-kubectl rollout status statefulset dev-db-mongodb-primary
-kubectl rollout status statefulset dev-db-mongodb-secondary
+mongodb_replicaset() {
+    ### TODO:  update to use chart in project
+    kubectl create configmap \
+        --from-file=${DIR}/configs/mongodb_init_scripts mongodb-init-scripts \
+        --dry-run=client -o yaml | kubectl apply -f -
+
+    helm upgrade --install dev-db bitnami/mongodb \
+        --version $BITNAMI_MONGODB_VER \
+        -f "${DIR}/configs/mongodb_options.yaml"  \
+        --set "volumePermissions.enabled=true" \
+        --set "persistence.storageClass=standard" \
+        --set "usePassword=true" \
+        --set "mongodbRootPassword=$MONGODB_ROOT_PASSWORD" \
+        --set "replicaSet.key=$MONGODB_RS_KEY" \
+        --set "extraEnvVars[0].name=MONGODB_APP_USERNAME" \
+        --set "extraEnvVars[0].value=$MONGODB_APP_USERNAME" \
+        --set "extraEnvVars[1].name=MONGODB_APP_PASSWORD" \
+        --set "extraEnvVars[1].value=$MONGODB_APP_PASSWORD" \
+        --set "extraEnvVars[2].name=MONGODB_APP_DATABASE" \
+        --set "extraEnvVars[2].value=$MONGODB_APP_DATABASE" \
+        --set "extraEnvVars[3].name=MONGODB_NAMESPACE" \
+        --set "extraEnvVars[3].value=default" \
+        --set "replicaSet.pdb.minAvailable.secondary=1" \
+        --set "replicaSet.pdb.minAvailable.arbiter=0" \
+        --set "replicaSet.replicas.secondary=0" \
+        --set "replicaSet.replicas.arbiter=0"
+
+    kubectl rollout status statefulset dev-db-mongodb-primary
+    kubectl rollout status statefulset dev-db-mongodb-secondary
+}
+
+mongodb_sharded() {
+    local CHART_PATH="$(dirname $0)/../../../../solution-base/mongodb/charts/mongodb-sharded"
```

Comment:
Any way to use the the manifest generated for the solution instead, to ensure it works?

Reply:
that may be a bit difficult as the manifest generated for the solution uses the metalk8s registry.

Reply:
If the only difference is the image (or any other param that kubectl can patch), it can be easily patched prior to applying with something like this:
```shell
echo "resources: [ $GENERATED_YAML_FILE ]" > kustomization.yaml
kustomize edit set image mongodb=$(get full image source with tag from deps.yaml) mongodb-exporter=$(...) mongodb-minideb=$(...) ...
kubectl apply -k .
```